api_client/python/timesketch_api_client/scenario.py

```diff
@@ -55,8 +58,67 @@ def name(self):
         Returns:
             Scenario name as string.
         """
+        if self._name:
+            return self._name
         scenario = self.lazyload_data()
-        return scenario["objects"][0]["name"]
+        self._name = scenario["objects"][0]["name"]
+        return self._name
+
+    @property
+    def display_name(self):
+        """Property that returns the scenario display name."""
+        if self._display_name:
+            return self._display_name
+        scenario = self.lazyload_data()
+        self._display_name = scenario["objects"][0]["display_name"]
+        return self._display_name
+
+    @display_name.setter
+    def display_name(self, value):
+        """Sets the display name of the scenario."""
+        resource_url = f"{self.api.api_root}/{self.resource_uri}"
+        data = {"scenario_name": value}
+        response = self.api.session.post(resource_url, json=data)
+        self._data = error.get_response_json(response, logger)
+        updated_object = self._data["objects"][0]
+        self._display_name = updated_object["display_name"]
+        self._name = updated_object["name"]
```

Comment:
Why is the name here updated? Isn't that method about the display name?

Reply:
Good catch, this is not necessary.